### PR TITLE
Add Debrief from 2021 Robocup

### PR DESCRIPTION
**Pull request by LachlanCourt**
[Preview](https://deploy-preview-149--nubook.netlify.app/team/robocup/debriefs)

### CODE REVIEW

File: src/book/01-team/05-robocup/03-debriefs.mdx [hunk 1]
```diff
@@ -6,9 +6,171 @@ description: Debriefs and takeaways from our performances at RoboCup.
...
+- Have a look at implementing ROS bridge, a system with a huge number of libraries that we can use if we can interface with it
+- Investigate PlotJuggler to be used as potential alternative to NUsight charts
+- Numerical optimization for Inverse Kinematics
+- Highly accurate system identification so our model best translates from Webots to the real world
+- Path planning/tracking (vector fields that include obstacles or something similar)
+- Develop a simple program that uses sliders for tuning YAML files for our scripts
+- Multi-state walk engine, which starts in a slower state where it is more stable, then once it has started moving gradually increment the speed
+- Behaviour visualisation. Either some indication in existing NUsight views of the state of the robot, or a new NUsight behaviour view, with some indication of graphs of previous states and possible future states.
+- Potentially a different NUsight view for different providers e.g. a NUsight view for the walk engine behaviour provider
+- Change `GameController.cpp` to use `Single` instead of `Sync` - `Single` can drop messages, but `Sync` will not
```
Comment (ysims) [inline src/book/01-team/05-robocup/03-debriefs.mdx:164]: Wait is this right, I thought it was the other way around. And we want whatever does not drop messages?

Comment (ysims) [reply src/book/01-team/05-robocup/03-debriefs.mdx:164]: @TrentHouliston @Bidski

File: src/book/01-team/05-robocup/03-debriefs.mdx [hunk 2]
```diff
@@ -6,9 +6,171 @@ description: Debriefs and takeaways from our performances at RoboCup.
...
+- Fix `BallDetector` to properly let the system know if it can detect no balls
+- Set up a different config for our playing role, so we debug and test on one role, and play on the role for playing - to prevent breaking config changes like the network traffic filter
```
Comment (ysims) [inline src/book/01-team/05-robocup/03-debriefs.mdx:166]: 'debug and test on one set of configs and play on another set of configs' sounds more like what this is about?

File: src/book/01-team/05-robocup/03-debriefs.mdx (hunk 1 above)
Comment (LachlanCourt) [reply src/book/01-team/05-robocup/03-debriefs.mdx:164]: Pretty sure this was straight from the google doc I just added the formatting

Comment (LachlanCourt) [reply src/book/01-team/05-robocup/03-debriefs.mdx:164]: File currently has one `single` and no `sync` so I think it must be the other way round

**Review by ysims — APPROVED** [review]
LGTM

**Pull request merged**